<?xml version="1.0" encoding="utf-8"?>
<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
  <file datatype="xml" original="markdown" source-language="en-US" target-language="nl-nl">
    <header>
      <tool tool-id="mdxliff" tool-name="mdxliff" tool-version="1.0-192e1fd" tool-company="Microsoft" />
      <xliffext:olfilehash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">a99f6363a055cbd8272c7514eb695af7843d28b2</xliffext:olfilehash>
      <xliffext:olfilepath xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset\System.ContextMarshalException.yml</xliffext:olfilepath>
      <xliffext:oltranslationpriority xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">fulldocset</xliffext:oltranslationpriority>
      <xliffext:oltranslationtype xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">Human Translation</xliffext:oltranslationtype>
      <xliffext:olskeletonhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">99c3297e0277268ff38951652df71f61d6bd78ad</xliffext:olskeletonhash>
      <xliffext:olxliffhash xmlns:xliffext="urn:microsoft:content:schema:xliffextensions">24008431dee4e83f1d771ae3553ecbbea209b0c8</xliffext:olxliffhash>
    </header>
    <body>
      <group id="content" extype="content">
        <trans-unit id="101" translate="yes" xml:space="preserve">
          <source>The exception that is thrown when an attempt to marshal an object across a context boundary fails.</source>
          <target state="translated">De uitzondering die wordt gegenereerd wanneer een poging tot een object door een contextgrens te marshallen mislukt.</target>       </trans-unit>
        <trans-unit id="102" translate="yes" xml:space="preserve" extradata="MT">
          <source>Objects can marshal by value or by reference.</source>
          <target state="translated">Objecten kunnen het marshallen waarde of door middel van verwijzing.</target>       </trans-unit>
        <trans-unit id="103" translate="yes" xml:space="preserve" extradata="MT">
          <source>Any attempt to pass a type that cannot be marshaled through a context boundary will result in a ContextMarshalException.</source>
          <target state="translated">Elke poging om door te geven van een type dat door een contextgrens kan niet worden gemarshald leidt tot een ContextMarshalException.</target>       </trans-unit>
        <trans-unit id="104" translate="yes" xml:space="preserve" extradata="MT">
          <source>ContextMarshalException uses the HRESULT COR_E_CONTEXTMARSHAL which has the value 0x80131504L.</source>
          <target state="translated">ContextMarshalException maakt gebruik van de HRESULT COR_E_CONTEXTMARSHAL waarvoor de waarde 0x80131504L.</target>       </trans-unit>
        <trans-unit id="105" translate="yes" xml:space="preserve" extradata="MT">
          <source>ContextMarshalException uses the default &lt;xref:System.Object.Equals%2A&gt; implementation, which supports reference equality.</source>
          <target state="translated">ContextMarshalException gebruikt de standaardinstallatielocatie &lt;xref:System.Object.Equals%2A&gt;implementatie, die ondersteuning biedt voor verwijzingsgelijkheid.&lt;/xref:System.Object.Equals%2A&gt;</target>       </trans-unit>
        <trans-unit id="106" translate="yes" xml:space="preserve" extradata="MT">
          <source>For a list of initial property values for an instance of ContextMarshalException, see the &lt;xref:System.ContextMarshalException.%23ctor%2A&gt; constructors.</source>
          <target state="translated">Zie voor een lijst met eerste eigenschapswaarden voor een exemplaar van ContextMarshalException de &lt;xref:System.ContextMarshalException.%23ctor%2A&gt;constructors.&lt;/xref:System.ContextMarshalException.%23ctor%2A&gt;</target>       </trans-unit>
        <trans-unit id="107" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with default properties.</source>
          <target state="translated">Initialiseert een nieuw exemplaar van de <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse met de standaardeigenschappen.</target>       </trans-unit>
        <trans-unit id="108" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ContextMarshalException&gt;.</source>
          <target state="translated">De volgende tabel toont de eerste waarden voor een exemplaar van &lt;xref:System.ContextMarshalException&gt;.&lt;/xref:System.ContextMarshalException&gt;</target>       </trans-unit>
        <trans-unit id="109" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|The empty string ("").|</source>
          <target state="translated">| Eigenschap | Waarde |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| <ph id="ph1">`null`</ph>.|   | &lt;xref:System.Exception.Message%2A&gt;| De lege tekenreeks (""). |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="110" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message.</source>
          <target state="translated">Initialiseert een nieuw exemplaar van de <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse met een opgegeven foutbericht.</target>       </trans-unit>
        <trans-unit id="111" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ContextMarshalException&gt;.</source>
          <target state="translated">De volgende tabel toont de eerste waarden voor een exemplaar van &lt;xref:System.ContextMarshalException&gt;.&lt;/xref:System.ContextMarshalException&gt;</target>       </trans-unit>
        <trans-unit id="112" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|<ph id="ph1">`null`</ph>.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Eigenschap | Waarde |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| <ph id="ph1">`null`</ph>.|   | &lt;xref:System.Exception.Message%2A&gt;| De fouttekenreeks bericht. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="113" translate="yes" xml:space="preserve">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Het foutbericht met uitleg over de reden voor de uitzondering.</target>       </trans-unit>
        <trans-unit id="114" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with serialized data.</source>
          <target state="translated">Initialiseert een nieuw exemplaar van de <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse met de geserialiseerde gegevens.</target>       </trans-unit>
        <trans-unit id="115" translate="yes" xml:space="preserve">
          <source>The <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> that holds the serialized object data about the exception being thrown.</source>
          <target state="translated">De <bpt id="p1">&lt;xref href="System.Runtime.Serialization.SerializationInfo"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> waarin zich de geserialiseerd objectgegevens over de uitzondering.</target>       </trans-unit>
        <trans-unit id="116" translate="yes" xml:space="preserve">
          <source>The &lt;xref:System.Runtime.Serialization.StreamingContext&gt; that contains contextual information about the source or destination.</source>
          <target state="translated">De &lt;xref:System.Runtime.Serialization.StreamingContext&gt;bevat contextuele informatie over de bron- of doelserver.&lt;/xref:System.Runtime.Serialization.StreamingContext&gt;</target>       </trans-unit>
        <trans-unit id="117" translate="yes" xml:space="preserve">
          <source>Initializes a new instance of the <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept> class with a specified error message and a reference to the inner exception that is the cause of this exception.</source>
          <target state="translated">Initialiseert een nieuw exemplaar van de <bpt id="p1">&lt;xref href="System.ContextMarshalException"&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept> klasse met een opgegeven foutbericht en een verwijzing naar de interne uitzondering dat de oorzaak van deze uitzondering.</target>       </trans-unit>
        <trans-unit id="118" translate="yes" xml:space="preserve" extradata="MT">
          <source>An exception that is thrown as a direct result of a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.</source>
          <target state="translated">Een uitzondering die is gegenereerd als direct resultaat van een eerdere uitzondering moet een verwijzing bevatten naar de vorige uitzondering in de &lt;xref:System.Exception.InnerException%2A&gt;eigenschap.&lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="119" translate="yes" xml:space="preserve" extradata="MT">
          <source>The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor, or <ph id="ph1">`null`</ph> if the &lt;xref:System.Exception.InnerException%2A&gt; property does not supply the inner exception value to the constructor.</source>
          <target state="translated">De &lt;xref:System.Exception.InnerException%2A&gt;eigenschap dezelfde waarde die is doorgegeven aan de constructor of <ph id="ph1">`null`</ph> als de &lt;xref:System.Exception.InnerException%2A&gt;eigenschap levert niet de waarde van de binnenste uitzondering voor de constructor.&lt;/xref:System.Exception.InnerException%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="120" translate="yes" xml:space="preserve" extradata="MT">
          <source>The following table shows the initial property values for an instance of &lt;xref:System.ContextMarshalException&gt;.</source>
          <target state="translated">De volgende tabel toont de eerste waarden voor een exemplaar van &lt;xref:System.ContextMarshalException&gt;.&lt;/xref:System.ContextMarshalException&gt;</target>       </trans-unit>
        <trans-unit id="121" translate="yes" xml:space="preserve" extradata="MT">
          <source>|Property|Value|   |--------------|-----------|   |&lt;xref:System.Exception.InnerException%2A&gt;|The inner exception reference.|   |&lt;xref:System.Exception.Message%2A&gt;|The error message string.|</source>
          <target state="translated">| Eigenschap | Waarde |   |--------------|-----------|   | &lt;xref:System.Exception.InnerException%2A&gt;| De verwijzing naar de interne uitzondering. |   | &lt;xref:System.Exception.Message%2A&gt;| De fouttekenreeks bericht. |&lt;/xref:System.Exception.Message%2A&gt; &lt;/xref:System.Exception.InnerException%2A&gt;</target>       </trans-unit>
        <trans-unit id="122" translate="yes" xml:space="preserve">
          <source>The error message that explains the reason for the exception.</source>
          <target state="translated">Het foutbericht met uitleg over de reden voor de uitzondering.</target>       </trans-unit>
        <trans-unit id="123" translate="yes" xml:space="preserve">
          <source>The exception that is the cause of the current exception.</source>
          <target state="translated">De uitzondering die de oorzaak van de huidige uitzondering.</target>       </trans-unit>
        <trans-unit id="124" translate="yes" xml:space="preserve">
          <source>If the <ph id="ph1">`inner`</ph> parameter is not <bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt><ept id="p1">&lt;/xref&gt;</ept>, the current exception is raised in a <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt><ept id="p2">&lt;/xref&gt;</ept> block that handles the inner exception.</source>
          <target state="translated">Als de <ph id="ph1">`inner`</ph> parameter is niet <bpt id="p1">&lt;xref uid="langword_csharp_null" name="null" href=""&gt;</bpt> <ept id="p1">&lt;/xref&gt;</ept>, de huidige uitzondering is opgetreden tijdens een <bpt id="p2">&lt;xref uid="langword_csharp_catch" name="catch" href=""&gt;</bpt> <ept id="p2">&lt;/xref&gt;</ept> blok die verantwoordelijk is voor de interne uitzondering.</target>       </trans-unit>
      </group>
    </body>
  </file>
</xliff>